examples/25-Chronos-2-examples.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    plt.style.use("ggplot")
</pre><p>Let's use the default style to comply with our other example notebooks. This will mean, that the order of plotting the series will have to be adjusted a bit.</p><p> </p><p>Usually, we plot the actual series first (black color), and then the predictions (blues, purples, etc.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2944/discussion/#comment-2531865938'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>